spec/season_statistics_spec.rb

```diff
@@ -0,0 +1,205 @@
+require 'spec_helper'
+
+RSpec.describe SeasonStats do
+   before(:each) do
+      game_path = './data/games_fixture.csv'
+      team_path = './data/teams_fixture.csv'
+      game_teams_path = './data/game_teams_fixture.csv'
+
+      locations = {
+         games: game_path,
+         teams: team_path,
+         game_teams: game_teams_path
+      }
+
+      @stat_tracker = StatTracker.from_csv(locations)
```

Comment:
This is really great work! Just for clarification, we have to create a new instance of StatTracker to each test in order to test each class's methods?